### DISCUSSION TITLE
Paperless dont Work after conf change

### DISCUSSION THREAD

**Discussion by Missenberger79** [tteck/Proxmox#2148] [Help Needed]
Hi, after i Change Something in the paperless.conf nothing working f.e. import a new documemt.
After i start the Script again everything working again.
Ist this normal or what did i wrong.
Thanks

**Comment by tteck** [comment]
Typically, before making changes you would stop Paperless-ngx 
```
systemctl stop paperless-consumer paperless-webserver paperless-scheduler paperless-task-queue.service
```
make changes, then start everything up
```
systemctl start paperless-consumer paperless-webserver paperless-scheduler paperless-task-queue.service
```
